### DISCUSSION TITLE
Unwrap wysiwyg display template

### DISCUSSION THREAD

**Discussion by ArtemAstakhov** [directus/directus#15400] [Feature Requests]
When the wysiwyg field is used as a Display template it is wrapped with html tags
would be good to unwrap it 

![изображение](https://user-images.githubusercontent.com/18442991/188384621-effd3075-ddc1-47ca-aeb8-9a874d946f3b.png)

**Comment by rijkvanzanten** [comment]
If no display is configured, the app will fall back to the raw value saved in the database. In this case, you might wanna configure the _Description_ field to use the _Formatted Value_ display. That will render it without the HTML tags 🙂

**Reply by ArtemAstakhov** [reply]
thank you, Formatted Value works 🎉

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
